src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -0,0 +1,54 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Entity class for money wallet.
+ *
+ * @author DawidStuzynski
+ * @version 1.0
+ * @since track-expenses-app-backend 2.0
+ */
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class Wallet implements Serializable {
+
+    /**
+     * Class version 1.0 -> 100010L
+     */
+    @Serial
+    private static final long serialVersionUID = 100010L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Setter
+    @NotBlank
+    @Size(max = 20)
+    @Pattern(regexp = "[a-z A-Z]+")
+    private String name;
+
+    private Instant creationDate;
+
```

Comment:
Could you please write why you decided to use "Instant" type? Why do you think this is a the best solution? In the feature I think about batch for the report. But it will be much later.

Reply:
@Iwona007 You can look around here. https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html 
I think, that's good option to use 'Instant' type.

Reply:
@Iwona007 I think, 'Instant' is correct type here. You can look around here if you want. https://stackoverflow.com/questions/32437550/whats-the-difference-between-instant-and-localdatetime